src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/CoreCLR/WebRequestPSCmdlet.CoreClr.cs

```diff
@@ -389,6 +389,12 @@ internal virtual void FillRequestStream(HttpRequestMessage request)
                     byte[] bytes = content as byte[];
                     SetRequestContent(request, bytes);
                 }
+                else if (content is MultipartFormDataContent)
+                {
+                    WebSession.ContentHeaders.Clear();
+                    MultipartFormDataContent multipartFormDataContent = content as MultipartFormDataContent;
```

Comment:
We should exclude double casting. Here it is better use C# 7.0 pattern:
```c#
else if (content is MultipartFormDataContent multipartFormDataContent ) 
```

Reply:
Awesome. I will do that.